### PR TITLE
[bug 1102290] Add test for contribute.json.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/humans/tests/test_views.py [hunk 1]
```diff
@@ -13,3 +15,12 @@ def test_base(self):
...
         eq_(response.status_code, 302)
         ok_(settings.HUMANSTXT_URL in response._headers['location'][1])
+
+
+class TestContribute(TestCase):
+    def test_base(self):
+        client = Client()
+        response = client.get(reverse('humans:contribute-view'))
+        eq_(response.status_code, 200)
+        ok_(json.loads(response.content))
```
Comment (johngian) [inline mozillians/humans/tests/test_views.py]: You don't need to parse ``contribute.json`` in order to check ``response['Content-Type']``.

File: mozillians/humans/tests/test_views.py [hunk 2]
```diff
@@ -13,3 +15,12 @@ def test_base(self):
...
+        eq_(response['Content-Type'], 'application/json')
```
Comment (johngian) [inline mozillians/humans/tests/test_views.py]: Check also please that the correct template is being rendered.

**Comment by johngian** [comment]
r+wc :rocket:

File: mozillians/humans/tests/test_views.py (hunk 1 above)
Comment (akatsoulas) [inline mozillians/humans/tests/test_views.py]: I am using this to test if this is a valid JSON.

**Pull request merged**